src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -205,6 +205,15 @@ public static GitHub connectAnonymously() throws IOException {
     }
 
     /**
+     * Connects to GitHub Enterprise anonymously.
```

Comment:
What is the difference between anonymous and not? Why library should care about specific logins?